awswrangler/typing.py

```diff
@@ -165,6 +165,29 @@ class AthenaPartitionProjectionSettings(TypedDict):
     """
 
 
+class RayModinSettings(TypedDict):
```

Comment:
Should we just call it `RaySettings`. These arguments are used to read Ray datasets independently of Modin and we might have another memory format in the future that would benefit from them

Reply:
+1